agent/eni/networkutils/utils_windows_test.go

```diff
@@ -237,3 +254,39 @@ func TestGetDNSServerAddressList(t *testing.T) {
 	assert.Len(t, dnsServerList, 1)
 	assert.EqualValues(t, dnsServerList[0], validDnsServer)
 }
+
+// TestGetMIBIfEntryFromDeviceName tests the GetMIBIfEntryFromDeviceName method.
+func TestGetMIBIfEntryFromDeviceName(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	funcGetIfTable2Ex = func() (*mibIfTable2, error) {
+		return mockMibIfTable2, nil
+	}
+	funcFreeMibTable = func(table **mibIfTable2) {
+		// Empty function.
+		return
+	}
+
+	netUtils := &networkUtils{}
+	ifRow, err := netUtils.GetMIBIfEntryFromDeviceName(deviceName)
+	assert.NoError(t, err)
+	assert.Equal(t, uint32(interfaceIndex), ifRow.interfaceIndex)
+	assert.Equal(t, ifaceLUID, ifRow.InterfaceLUID)
+}
+
+// TestGetMIBIfEntryFromLUID tests the GetMIBIfEntryFromLUID method.
+func TestGetMIBIfEntryFromLUID(t *testing.T) {
```

Comment:
Since we went to all the trouble here lets add the negative test if we return error from the call too.